examples/demo-stf/src/batch_builder.rs

```diff
@@ -0,0 +1,332 @@
+use anyhow::bail;
+use borsh::BorshDeserialize;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_api::{Context, DispatchCall, PublicKey, Spec};
+use sov_rollup_interface::services::batch_builder::BatchBuilder;
+use sov_state::WorkingSet;
+use std::cell::RefCell;
+use std::collections::VecDeque;
+use std::io::Cursor;
+
+/// BatchBuilder that creates batch of transaction in the order they were submitted
+/// Only transaction that were successfully dispatched are included.
+pub struct FiFoStrictBatchBuilder<R, C: Context> {
+    mempool: RefCell<VecDeque<Vec<u8>>>,
```

Comment:
Why do we need the `RefCell ` here?  Is it possible to take `accept_tx  & get_next_blob ` to take &mut self?